client/layout/guided-tours/tours/checklist-about-page-tour.js

```diff
@@ -21,6 +21,9 @@ import {
 	Tour,
 } from 'layout/guided-tours/config-elements';
 
+const SetFeaturedImageButtonLabel = translate( 'Set Featured Image' );
+const UpdateButtonLabel = translate( 'Update' );
```

Comment:
Note that by translating these in the module-global scope they won't be updated on language changes. It's usually our practice to translate them on render so that they will always be up-to-date. In cases like this if we need to we can turn them into constant-producing functions and call them later as functions instead of constants.

```js
const SetFeaturedImageButtonLabel = () => translate( 'Set Featured Image' );

…

SetFeaturedImageButtonLabel()
```

This would also be a good time to update this component so that instead of using the imported `translate` we instead wrap it in the `localize()` wrapper and use `this.props.translate()`…

```js
import { localize } from 'i18n-calypso';

…

export const ChecklistAboutPageTour = … {
	const { translate } = this.props
}

export default localize( ChecklistAboutPageTour );
```

though I'm not sure if this is possible given the `makeTour()` function which I know nothing about :wink: